NGitLab/Models/UserQuery.cs

```diff
@@ -0,0 +1,50 @@
+﻿namespace NGitLab
+{
+    public class UserQuery
+    {
+        /// <summary>
+        /// Specifies how many record per paging (Gitlab supports a maximum of 100 projects and defaults to 20).
+        /// </summary>
+        public int? PerPage;
```

Comment:
Idem for other properties

```suggestion
        public int? PerPage { get; set;} 
```

Reply:
Updated